package.json

```diff
@@ -41,6 +41,13 @@
     "test-ui": "TZ=UTC vitest --ui"
   },
   "dependencies": {
+    "@codemirror/autocomplete": "^6.3.4",
```

Comment:
Should we be using both Codemirror _and_ Prism? Also: is there any reason we picked something like Codemirror and _not_ Monaco?

Reply:
Considering we're looking at Visual Studio Code extensions, there _might_ be an argument for Monaco.

Reply:
Prism doesn't have an editor afaik, and we wanted to provide a richer experience for writing JSON than just a `<textarea />`. Monaco does look promising though, I hadn't heard of it.

Reply:
Yea, Monaco is just VS Code's editor.